test/quic_multistream_test.c

```diff
@@ -3025,6 +3025,65 @@ static const struct script_op script_41[] = {
     OP_END
 };
 
+/* 42. Fault injection - CRYPTO frame with illegal offset */
+static int script_42_inject_plain(struct helper *h, QUIC_PKT_HDR *hdr,
+                                  unsigned char *buf, size_t len)
+{
+    int ok = 0;
+    unsigned char frame_buf[64];
+    size_t written;
+    WPACKET wpkt;
+
+    if (h->inject_word0 == 0)
+        return 1;
+
+    --h->inject_word0;
+
+    if (!TEST_true(WPACKET_init_static_len(&wpkt, frame_buf,
+                                           sizeof(frame_buf), 0)))
+        return 0;
+
+    if (!TEST_true(WPACKET_quic_write_vlint(&wpkt, OSSL_QUIC_FRAME_TYPE_CRYPTO))
+        || !TEST_true(WPACKET_quic_write_vlint(&wpkt, (((uint64_t)1) << 62) - 1))
+        || !TEST_true(WPACKET_quic_write_vlint(&wpkt, 1))
```

Comment:
I wonder if additionally checking maximum offset *and* maximum length would be worthwhile? i.e. to check no overflows etc.

Reply:
This will come in the Assorted Tests PR.